python/reg_interface/scripts/repeated_reg_read.py

```diff
@@ -0,0 +1,116 @@
+    bitstuffing_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.BITSTUFFING_ERROR_CNT").real_address)
+    timeout_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.TIMEOUT_ERROR_CNT").real_address)
+    axi_strobe_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.AXI_STROBE_ERROR_CNT").real_address)
+    transaction_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.TRANSACTION_CNT").real_address)
+
+    print >> f, "| after  | "+crc_error_cnt_after+"    | "+packet_error_cnt_after+"       | "+bitstuffing_error_cnt_after+"            | "+timeout_error_cnt_after+"        | "+axi_strobe_error_cnt_after+"           | "+transaction_cnt_after+"      |"
+
+    print >> f, "| delta  | "+str(int(crc_error_cnt_after,16) - int(crc_error_cnt_before,16))+"             | "+str(int(packet_error_cnt_after,16) - int(packet_error_cnt_before,16))+"                | "+str(int(bitstuffing_error_cnt_after,16) - int(bitstuffing_error_cnt_before,16))+"                     | "+str(int(timeout_error_cnt_after,16) - int(timeout_error_cnt_before,16))+"                 | "+str(int(axi_strobe_error_cnt_after,16) - int(axi_strobe_error_cnt_before,16))+"               | "+str(int(transaction_cnt_after,16) - int(transaction_cnt_before,16))+"          |"    
+    
+#    print "GEM_AMC.SLOW_CONTROL.VFAT3.CRC_ERROR_CNT = "+crc_error_cnt_after
+#    print "GEM_AMC.SLOW_CONTROL.VFAT3.PACKET_ERROR_CNT = "+packet_error_cnt_after
+#    print "GEM_AMC.SLOW_CONTROL.VFAT3.BITSTUFFING_ERROR_CNT = "+ bitstuffing_error_cnt_after
+#    print "GEM_AMC.SLOW_CONTROL.VFAT3.TIMEOUT_ERROR_CNT = "+timeout_error_cnt_after
+#    print "GEM_AMC.SLOW_CONTROL.VFAT3.AXI_STROBE_ERROR_CNT = "+axi_strobe_error_cnt_after
+#    print "GEM_AMC.SLOW_CONTROL.VFAT3.TRANSACTION_CNT = "+transaction_cnt_after
+#    print "GEM_AMC.GEM_SYSTEM.CTRL.LINK_RESET = "+link_reset_after
+
+#    print "Delta GEM_AMC.SLOW_CONTROL.VFAT3.CRC_ERROR_CNT = "+str(int(crc_error_cnt_after,16) - int(crc_error_cnt_before,16))
+#    print "Delta GEM_AMC.SLOW_CONTROL.VFAT3.PACKET_ERROR_CNT = "+str(int(packet_error_cnt_after,16) - int(packet_error_cnt_before,16))
+#    print "Delta GEM_AMC.SLOW_CONTROL.VFAT3.BITSTUFFING_ERROR_CNT = "+ str(int(bitstuffing_error_cnt_after,16) - int(bitstuffing_error_cnt_before,16)) 
+#    print "Delta GEM_AMC.SLOW_CONTROL.VFAT3.TIMEOUT_ERROR_CNT = "+str(int(timeout_error_cnt_after,16) - int(timeout_error_cnt_before,16))
+#    print "Delta GEM_AMC.SLOW_CONTROL.VFAT3.AXI_STROBE_ERROR_CNT = "+str(int(axi_strobe_error_cnt_after,16) - int(axi_strobe_error_cnt_before,16))
+#    print "Delta GEM_AMC.SLOW_CONTROL.VFAT3.TRANSACTION_CNT = "+str(int(transaction_cnt_after,16) - int(transaction_cnt_before,16))
+#    print "Delta GEM_AMC.GEM_SYSTEM.CTRL.LINK_RESET = "+str(int(link_reset_after,16) - int(link_reset_before,16))
+
+    print >> f, d
+    
+    #readAddress()
```

Comment:
What is the purpose of these commented code blocks?  Typically we should avoid pushing code with large commented sections into the central repo.